nisapi/clean/helpers.py

```diff
@@ -0,0 +1,309 @@
+import polars as pl
+import uuid
+from typing import Sequence
+
+"""Data schema to be used for all datasets"""
+data_schema = pl.Schema(
+    [
+        ("vaccine", pl.String),
+        ("geographic_type", pl.String),
+        ("geographic_value", pl.String),
+        ("demographic_type", pl.String),
+        ("demographic_value", pl.String),
+        ("indicator_type", pl.String),
+        ("indicator_value", pl.String),
+        ("week_ending", pl.Date),
```

Comment:
Looks like this is not up-to-date with the newest schema that includes time period type, begin and end dates, and full confidence intervals (which I suppose is my fault for being slow to review this).

Reply:
This will be updated in #35